### DISCUSSION TITLE
Anyone had success setting up nvim with the built-in bash of cmder?

### DISCUSSION THREAD

**Discussion by rlegendi** [cmderdev/cmder#2826] [Q&A]
Hi All,

I'd love to use neovim with cmder. Recently I was experimenting with wsl to make it work, which fixed a couple of problems (like gx functionality), but a lot of other things got broke (including using registers to copy paste).

**Did anyone have any luck with running nvim under cmder with the built-in bash settings?**

E.g., one of the functionalities that is missing is when you simply open up neovim, type `!!date`. It should execute the `date` command through the shell and put the output into the text being edited.

I usually got some temp file access errors, but I guess the reason being the terminal commands cannot be run properly. I was experimenting with different settings for `:set shell=...` but with no luck.

Any hints are more than welcome!

**Comment by zbyna** [comment]
There are bunch of bugs related this:
https://github.com/neovim/neovim/issues/14605
https://github.com/neovim/neovim/issues/16957

Anyway [this](https://github.com/neovim/neovim/issues/14605#issuecomment-1417532544) temporary solution works for me:

```:let &shell = '"C:\WINDOWS\system32\cmd.exe"' ``` in your `~/AppData/Local/nvim/init.vim`
